tests/unit/post_processors/conftest.py

```
@@ -62,6 +65,211 @@
+            self.export_calls.append((metrics_data, timeout_millis, kwargs))
+            return FakeMetricExportResult.SUCCESS
+
+        def force_flush(self, timeout_millis: int = 30000) -> bool:
+            self.force_flush_calls.append(timeout_millis)
+            return True
+
+        def shutdown(self, timeout_millis: float = 30000, **kwargs: object) -> None:
+            self.shutdown_calls += 1
+
+    class FakeOTLPMetricExporter(FakeMetricExporter):
+        instances: list["FakeOTLPMetricExporter"] = []
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)